src/components/communitySettings/communityPlatforms/TcCommunityPlatforms.tsx

```
@@ -347,75 +226,31 @@ function TcCommunityPlatforms() {
-									}
-									variant="text"
-									onClick={() => handleReputationScoreModule()}
-								/>
-							</div>
-						}
-					/>
+					{MODULE_CONFIGS.map(moduleConfig => (
+						<TcCard
+							key={moduleConfig.name}
+							className="max-h-[6rem] min-h-[6rem] min-w-[10rem] max-w-[10rem] flex-grow"
+							children={
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)